javascript/features/nuwani_commands/communication_commands.test.js

```diff
@@ -110,6 +110,21 @@ describe('CommunicationCommands', (it, beforeEach, afterEach) => {
         assert.includes(result[0], 'https://discord.sa-mp.nl/');
     });
 
+    it('cmds', async (assert) => {
+        const result = await issueCommand(bot, commandManager, {
+            source: kCommandSource,
+            command: '!help',
+        });
+        
```

Comment:
nit: just have a single blank line, and remove the whitespace on lines 124 and 125 as well.

This looks good to me with the comment from #586 applied. Thank you!